video/external/src/backend.rs

```diff
@@ -28,7 +29,7 @@ enum ProxyOrStream {
 /// except for H.264, for which it uses an external decoder.
 pub struct ExternalVideoBackend {
     streams: SlotMap<VideoStreamHandle, ProxyOrStream>,
-    openh264_lib_filepath: Option<PathBuf>,
+    h264_codec: Option<H264Codec>,
```

Comment:
Yet again, maybe a name more specific to OpenH264?

Reply:
Done